Adafruit_FRAM_SPI.cpp

```diff
@@ -343,3 +344,36 @@ bool Adafruit_FRAM_SPI::setStatusRegister(uint8_t value) {
 void Adafruit_FRAM_SPI::setAddressSize(uint8_t nAddressSize) {
   _nAddressSizeBytes = nAddressSize;
 }
+
+/*!
+ *  @brief  Enters the FRAM's low power sleep mode
+ *  @return true if successful
+ */
+// WARNING: this method has not yet been validated
+bool Adafruit_FRAM_SPI::enter_low_power_mode(void) {
+  uint8_t cmd;
+
+  cmd = OPCODE_SLEEP;
+
+  return spi_dev->write(&cmd, 1);
+}
+
+/*!
+ *  @brief  exits the FRAM's low power sleep mode
+ *  @return true if successful
+ */
+// WARNING: this method has not yet been validated
+bool Adafruit_FRAM_SPI::exit_low_power_mode(void) {
+  uint8_t cmd;
+
+  // Returning to an normal operation from the SLEEP mode is carried out after
+  // tREC (Max 400 μs) time from the falling edge of CS
+  spi_dev->beginTransactionWithAssertingCS();
+  delayMicroseconds(300);
+  // It is possible to return CS to H level before tREC time. However, it
+  // is prohibited to bring down CS to L level again during tREC period.
+  spi_dev->endTransactionWithDeassertingCS();
+  delayMicroseconds(100);
+
+  return spi_dev->write(&cmd, 1);
```

Comment:
we don't really need to do spi write() here, cs along should be able to put chip out of sleep